test_regress/t/t_vpi_force.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python3
+# DESCRIPTION: Verilator: Verilog Test driver/expect definition
+#
+# Copyright 2025 by Wilson Snyder. This program is free software; you
+# can redistribute it and/or modify it under the terms of either the GNU
+# Lesser General Public License Version 3 or the Perl Artistic License
+# Version 2.0.
+# SPDX-License-Identifier: LGPL-3.0-only OR Artistic-2.0
+
+import vltest_bootstrap
+
+test.scenarios('simulator')
+
+test.compile(make_top_shell=False,
+             make_main=False,
+             make_pli=True,
+             verilator_flags2=["--main --exe --vpi --timing", test.pli_filename],
```

Comment:
`--binary` == `--main --exe --timing`.  Similar on others.

```suggestion
             verilator_flags2=["--binary --vpi", test.pli_filename],
```